### PR TITLE
Fix Typo ProjectSettings.xml, in Editor/Script Templates description

### PR DESCRIPTION
**Pull request by merriam**
typo.  Sorry, it bugged me.

### CODE REVIEW

**Comment by Calinou** [comment]
Please amend the commit message to something more meaningful such as "Fix typo in the ProjectSettings class documentation".

File: doc/classes/ProjectSettings.xml [hunk 1]
```diff
@@ -469,7 +469,7 @@
 			[b]Note:[/b] This option is experimental and meant to alleviate stutter experienced by some users. However, some users have experienced a Vsync framerate halving (e.g. from 60 FPS to 30 FPS) when using it.
 		</member>
 		<member name="editor/script_templates_search_path" type="String" setter="" getter="" default="&quot;res://script_templates&quot;">
-			Search path for project-specific script templates. Script templates will be search both in the editor-specific path and in this project-specific path.
+			Search path for project-specific script templates. Script templates will be searched both in the editor-specific path and in this project-specific path.
```
Comment (aaronfranke) [inline doc/classes/ProjectSettings.xml:472]: This reads weirdly, we may as well improve this sentence by using the active voice.

```suggestion
			Search path for project-specific script templates. Godot will search for script templates both in the editor-specific path and in this project-specific path.
```

**Review by aaronfranke — CHANGES_REQUESTED** [review]

**Review by aaronfranke — APPROVED** [review]
A [squash](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html) would be nice, but the changes look good.

**Comment by akien-mga** [comment]
Force pushed an update with the commits squashed and a meaningful commit message as requested.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.2.4.

**Pull request merged**